packages/paste-core/components/textarea/stories/textarea.stories.tsx

```diff
@@ -418,3 +421,298 @@ export const TextareaDisabledInsertBeforeAndAfterInverse = (): React.ReactNode =
 TextareaDisabledInsertBeforeAndAfterInverse.story = {
   name: 'Textarea - Disabled insert before and after inverse',
 };
+
+export const CustomizedTextarea: React.FC = () => {
+  const activeTheme = useTheme();
+  const uidTwo = useUID();
+  const uidTwoVariant = useUID();
+  const uidThree = useUID();
+  const uidThreeVariant = useUID();
+  return (
+    <CustomizationProvider
+      baseTheme="default"
+      theme={activeTheme}
```

Comment:
So when we display these stories for VRT we place them side by side, one in default and one in dark mode. 

The problem is the theme is decided at the storybook level, so the dark mode rendered version is using the default theme because we hard code the base theme to be default.

By grabbing the active theme from context, which is supplied by storybook, we can just provide that theme to the customization provider so each side is using the correct theme.